postgres/content.md

```diff
@@ -68,6 +68,10 @@ This optional environment variable can be used to define a different name for th
 
 This optional environment variable can be used to send arguments to `postgres initdb`. The value is a space separated string of arguments as `postgres initdb` would expect them. This is useful for adding functionality like data page checksums: `-e POSTGRES_INITDB_ARGS="--data-checksums"`.
 
+### `POSTGRES_XLOG_DIR`
```

Comment:
This should be `POSTGRES_INITDB_XLOGDIR`, right? (https://github.com/docker-library/postgres/blob/123aedc09bc067740fb28921c500ded83c018bd4/docker-entrypoint.sh#L41)